homeassistant/helpers/template.py

```diff
@@ -2156,6 +2170,9 @@ def warn_unsupported(*args, **kwargs):
         self.globals["is_state_attr"] = hassfunction(is_state_attr)
         self.globals["state_attr"] = hassfunction(state_attr)
         self.globals["states"] = AllStates(hass)
+        self.globals["is_nominal"] = hassfunction(is_nominal)
+        self.filters["is_nominal"] = pass_context(self.globals["is_nominal"])
+        self.tests["is_nominal"] = pass_eval_context(self.globals["is_nominal"])
```

Comment:
Think we should rename the test to just `nominal`. `is_nominal` is fine for the filter and the function but the test will read better without the `is_` at the front. Currently you will use this test like this:
```
{{ 'sensor.test' is is_nominal }}
{{ ['sensor.a', 'sensor.b'] | select('is_nominal') | list }}
```
This reads better and is more fitting with the other jinja tests:
```
{{ 'sensor.test' is nominal }}
{{ ['sensor.a', 'sensor.b'] | select('nominal') | list }}
```

Reply:
Only half true as there is an equally good counter example: `{{ is_nominal(states.sensor.test) }}`, which is in line with all other `is_x` filters: https://www.home-assistant.io/docs/configuration/templating/
Of course I agree that your shown jinja test syntax is sexier

Reply:
I'm okay with any verbiage as long as this PR gets through.  I constantly see people not understanding how to use 'unknown' and 'unavailable' on everything they check because they don't understand it.

Reply:
This
```
{{ is_nominal(states.sensor.test) }}
```
And this
```
{{ 'sensor.test' is is_nominal }}
```
Can both work simultaneously. You specify the name to use when you add it to filters, tests and globals. It does not have to be the same name.

Basically I'm saying change the above code to this:
```py
        self.globals["is_nominal"] = hassfunction(is_nominal)
        self.filters["is_nominal"] = pass_context(self.globals["is_nominal"])
        self.tests["nominal"] = pass_eval_context(self.globals["is_nominal"])
```
Then when used as a filter or a function it is `is_nominal` and when used as a test it is just `nominal`

Reply:
Just like how we did with `regex_match` and `regex_search`. When using them as a filter you do this:
```
{{ 'test' | regex_match('^te.*') }}
{{ 'test' | regex_search('^te) }}
```
https://github.com/home-assistant/core/blob/8faecae34d9b45914eb7413383a8134c73234461/homeassistant/helpers/template.py#L2016

When using them as a test its just `match` and `search`:
```
{{ 'test' is match('^te.*') }}
{{ 'test' is search('^te) }}
```
https://github.com/home-assistant/core/blob/8faecae34d9b45914eb7413383a8134c73234461/homeassistant/helpers/template.py#L2068-L2069

Reply:
What me to change it to use ```nominal``` for tests? Easy change just want to confirm. :-)

Reply:
Yes please do 👍